UserAgent/Home View/TopSitesView.swift

```diff
@@ -36,8 +36,7 @@ class TopSitesView: UIView {
 // MARK: - Private Implementation
 private extension TopSitesView {
     private func setup() {
-        profile.panelDataObservers.activityStream.refreshIfNeeded(forceTopSites: true)
-
+        profile.panelDataObservers.activityStream.refreshIfNeeded(forceTopSites: false)
```

Comment:
we don't need to force update every time. Every navigation in browser will mark the top-site state as "Dirty" which will propagate the refresh if required.